receiver/purefbreceiver/factory.go

```diff
@@ -45,9 +45,9 @@ func createDefaultConfig() component.Config {
 		HTTPClientSettings: confighttp.HTTPClientSettings{},
 		Settings: &Settings{
 			ReloadIntervals: &ReloadIntervals{
-				Array:   15 * time.Second,
-				Clients: 300 * time.Second,
-				Usage:   300 * time.Second,
+				Array:   5 * time.Minute,
```

Comment:
@chrroberts-pure, should this be 5 minutes as well, or should this be kept at 15 seconds?

Reply:
Array should be set to 15s